packages/sync/src/config/DataSyncConfig.ts

```diff
@@ -80,4 +80,6 @@ export interface DataSyncConfig {
    * OpenShift specific configuration
    */
   openShiftConfig?: ConfigurationService;
+
+  resolvers?: IResolver;
```

Comment:
What is that? Missing docs.

Reply:
Ah.. resolvers are actual name for graphql resolvers on server.
Howe about rename this to conflictHandlers

Reply:
👍

Reply:
How about we still alow people to pass global and IResolver under the same name.
We can use pipe 

`ConflictResolutionStrategy | ConflictResolutionStrategies`